src/gql/user-search-query.js

```diff
@@ -0,0 +1,58 @@
+import gql from 'graphql-tag';
+
+const USER_SEARCH_QUERY = gql`
+  query UserSearchQuery(
+    $query: String!
+    $first: Int
+    $last: Int
+    $before: String
+    $after: String
+  ) {
+    search(
+      query: $query
+      type: USER
+      first: $first
+      last: $last
+      before: $before
+      after: $after
+    ) {
+      userCount
+      nodes {
+        ... on Organization {
+          avatarUrl
+          description
+          email
+          location
+          login
+          name
+          url
+        }
+        ... on User {
+          avatarUrl
+          email
+          followers {
+            totalCount
+          }
+          following {
+            totalCount
+          }
+          location
+          login
+          name
+          starredRepositories {
+            totalCount
+          }
+          url
+        }
```

Comment:
Want to talk with some other apollo/graphql folks about this - I couldn't fragment the shareable fields due to the typing differences even though they inherit from the same parent type. I feel like there should be a way to do this and if not, definitely a good future feature for the ecosystem.